docs/user-guide/creating-application/deployment-template/rollout-deployment.md

```diff
@@ -720,11 +720,15 @@ Wait for given period of time before scaling down the container.
 
 ## 3. Show Application Metrics
 
-If you want to see application metrics like different HTTP status codes metrics, application throughput, latency, response time. Enable the Application metrics from below the deployment template Save button. After enabling it, you should be able to see all metrics on App detail page. By default it remains disabled.
+If you want to see application metrics like different HTTP status codes metrics, application throughput, latency, response time, `show application metrics` option should be enabled. This can be enabled using the check box present below the deployment template.
+
+![Enable application metrics](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/creating-application/app-metrics/app-metrics-2.png)
+
+After enabling it, click on `save changes` and redeploy the application. Now you should be able to see all the metrics on app details page. By default it remains disabled.
```

Comment:
1. Enable application metrics
2. Click on `Save changes`
3. [Deploy the application](https://docs.devtron.ai/usage/applications/deploying-application/triggering-cd) on the respective environment.
4. After the deployment is successful, the application metrics should be visible on the [app detail page](https://docs.devtron.ai/usage/applications/app-details/debugging-deployment-and-monitoring#monitoring).

**NOTE: Pls replace direct links with file path.**